CHANGELOG.rst

```diff
@@ -1,7 +1,14 @@
 Changelog
 =========
 
-0.4.0 (2015-02-06)
+0.4.1 (2015-10-07)
+------------------
+* added notification config class to support custom text variables
+* allow disabling email html version
+* allow hiding of email body txt field
```

Comment:
txt is valid here as that's the extension of the email template that's modified by this field, so I added a better wording..